test/UnitTests/GitHub.App/ViewModels/GitHubPane/GitHubPaneViewModelTests.cs

```diff
@@ -23,150 +23,150 @@ public class TheInitializeMethod
         [Test]
         public async Task NotAGitRepositoryShownWhenNoRepository()
         {
-            var te = Substitute.For<ITeamExplorerServiceHolder>();
-            te.ActiveRepo.Returns(default(ILocalRepositoryModel));
-            var target = CreateTarget(teServiceHolder: te);
+            var te = CreateTeamExplorerContext();
+            te.ActiveRepository.Returns(default(ILocalRepositoryModel));
+            var target = CreateTarget(teamExplorerContext: te);
```

Comment:
Hmm this just seems to replace `Substitute.For<ITeamExplorerServiceHolder>()` with `CreateTeamExplorerContext()`, which isn't much different. my suggestion was to make `CreateTarget` accept a null `ITeamExplorerContext` by default and create the mock itself when the default mock is required.

Reply:
By the default mock do you mean `CreateTeamExplorerContext(ValidEnterpriseRepo)` rather than `CreateTeamExplorerContext(null)`?